picamera2/encoders/h264_encoder.py

```diff
@@ -13,10 +13,12 @@
 
 class H264Encoder(Encoder):
 
-    def __init__(self, bitrate):
+    def __init__(self, bitrate, repeat=False):
         super().__init__()
```

Comment:
We'll want a gop_size parameter too, but that's for a separate PR as it isn't related to this feature.

Reply:
Good point, just created an issue for that so I don't forget